plugin/externalPluginScript.go

```diff
@@ -4,16 +4,16 @@ import (
 	"bytes"
 	"context"
 	"fmt"
-	"os/exec"
-	"strings"
 
 	"github.com/puppetlabs/wash/activity"
+	"github.com/puppetlabs/wash/plugin/internal"
 )
 
 // externalPluginScript represents an external plugin's script
 type externalPluginScript interface {
 	Path() string
-	InvokeAndWait(ctx context.Context, args ...string) ([]byte, error)
+	InvokeAndWait(ctx context.Context, method string, entry *externalPluginEntry, args ...string) ([]byte, error)
```

Comment:
This specification mirrors the calling convention in the external plugin doc, which is why I wanted to do it.